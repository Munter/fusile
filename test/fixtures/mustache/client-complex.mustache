<p>Ok here we go with the local: {{ wow }}</p>
