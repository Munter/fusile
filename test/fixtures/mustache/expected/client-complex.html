<p>Ok here we go with the local: local</p>
